src/assets/scripts/client/navigationLibrary/StandardRoute/StandardRouteWaypointModel.js

```diff
@@ -142,32 +127,40 @@ export default class StandardRouteWaypointModel extends BaseModel {
         this._positionModel = null;
 
         /**
-         * Distance in nm from the previous waypoint.
+         * Any restrictions for a given fix
          *
-         * This property is set exterally by the `StandardRouteModel` and used only when called via
-         * `ArrivalBase.preSpawn()`.
+         * ex:
+         * - "A80+|S210"
```

Comment:
If we are now supporting restriction ranges, this doc block should be updated to include that